apps/site/lib/site_web/views/date_time_selector.ex

```diff
@@ -12,6 +12,26 @@ defmodule DateTimeSelector do
                     int -> Integer.to_string(int)
                   end)
 
+  @spec verify_params(map, map) :: map
```

Comment:
```suggestion
  @spec verify_params(Form.t(), DateTime.t()) :: map
```

I think these types might make sense for this function?